sources/schema.js

```diff
@@ -4,7 +4,8 @@ module.exports.datasources = {
       commitLength: {
         filter: {
           required: [
-            'repo'
+            'repo',
+            'owner'
```

Comment:
So to keep the progress we had prior to this merge, we should remove the `owner` from `required` because we want to refer to a repo as `author/name` instead of separating out both parts